13_microbiome_exploration.Rmd

```diff
@@ -159,6 +159,34 @@ plotRowTree(x[rowData(x)$Phylum %in% top_phyla_mean,],
             node_colour_by = "prevalence")
 ```
 
+## Abundance
+
+Another approach for exploring data is by visualizing abundance. `miaViz` offers
+the function `plotAbundanceDensity` where most abundant taxa can be plotted 
+including several options.
+
+In the following few demonstrations are shown, using the 
+[Lahti L et al. (2014)](https://doi.org/10.1038/ncomms5344) dataset.
+
+A Jitter plot using the top 25 taxa as a default value could be produced as follows: 
+
+```{r, warning=FALSE, message=FALSE}
+library(microbiomeDataSets)
+tse <- LahtiWAData()
+plotAbundanceDensity(tse, abund_values = "counts")
```

Comment:
Can you replace this one with 

plotAbundanceDensity(tse, layout = "jitter", abund_values = "relabundance", n = 40, point_size=1, point_shape=19, point_alpha=0.1) + scale_x_log10(label=scales::percent)

and then related to this also add citation to Fig S1 in [Salosensaari et al. 2021](https://www.nature.com/articles/s41467-021-22962-y) where we used a similar illustration.

Reply:
Sure, there seems btw to be some problem at the check related to installing dependencies.